test/TM1.jl

```diff
@@ -20,6 +20,9 @@ function check_containment(ftest, tma::T) where {T<:Union{TaylorModel1, RTaylorM
     return bb
 end
 
+interval_rand(X::Interval{T}) where {T} = X.lo + rand(T) * (X.hi - X.lo)
+interval_rand(X::IntervalBox) = interval_rand.(X)
```

Comment:
I think the methods of `interval_rand` are exactly equivalent to `rand(::Interval)` and `rand(::IntervalBox)`. If so, I think it is better to use directly those methods

Reply:
Thanks for the suggestion, I've replaced these functions with `rand`. I think it Is ready.